package.json

```diff
@@ -11,6 +11,7 @@
     "node-sass": "^7.0.1",
     "react": "^17.0.2",
     "react-bootstrap": "^2.1.1",
+    "react-code-blocks": "^0.0.9-0",
```

Comment:
I know this doesn't really matter but just bringing your attention to the fact that you added this dependency in the branch even though you haven't used it. Maybe you're planning on using it later though so just let me know

Reply:
I forgot about it, but want to display the answers.  I remember you showed me some link about react code blocks.  Don't remember where you sent it.  Displaying the code has been a pain in the butt.  What exactly should I be googling?

Reply:
Look at `react-code-blocks` documentation now that you already installed it I guess! https://www.npmjs.com/package/react-code-blocks